### DISCUSSION TITLE
Ability to Refresh the view without reopening the pane with calendar

### DISCUSSION THREAD

**Discussion by danieltomasz** [davish/obsidian-full-calendar#389] [Feature Requests]
## Delete this before submitting
Please include only one feature request per Issue. You can submit as many Issues as you have ideas. Thank you!

Currently the command "Open full calendar" reopen the pane with calendar and refresh the external  ical calendars.

Sometimes I have the pane resized and have view on specific day, using "open the calendar" reset to default view
Would be great if there would exist possibility to refresh the calendar in the background withot recreating all the elements of the view (via the palette command or button in calendar)

**Comment by bcohen44** [comment]
+1
